### PR TITLE
restyled news page

### LINKED ISSUE #714: ✨[Feature Request]: Rediesign the news page as per the current theme of the whole website
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

The current theme of the news page is not aligned with the theme of the website

### Use Case

Better UI helps in better customer response

### Benefits

_No response_

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by samarsajad**
## Description

Redesigned news page as per the theme of the website,added picture slider to make it attractive, added functionality to read more

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #714 

## Type of PR

- [x] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)

https://github.com/user-attachments/assets/2beab93c-71d8-4f50-90ed-d071cd54714e

## Checklist

- [x] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [x] I have updated my branch and synced it with project `main` branch before making this PR
- [x] I have performed a self-review of my code
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
GSSoC'24

### CODE REVIEW

**Comment by samarsajad** [comment]
Please review my pull request @varshith257 @Anishkagupta04

**Comment by samarsajad** [comment]
Please review my pull request @varshith257

**Comment by samarsajad** [comment]
Kindly take a look @varshith257

**Comment by varshith257** [comment]
@samarsajad Fix merge conflicts

**Comment by samarsajad** [comment]
> @samarsajad Fix merge conflicts

Done @varshith257

**Comment by varshith257** [comment]
@samarsajad Looks good to merge after fixing this suggestions:

Make sure all readme buttons
 be center at each card 

Duplicate navbars resulted. Fix it

**Comment by samarsajad** [comment]
> @samarsajad Looks good to merge after fixing this suggestions:
> 
> Make sure all readme buttons be center at each card
> 
> Duplicate navbars resulted. Fix it

I did a thorough screening of my code but I couldn't find anything related to duplicate nav bar. In fact in my local system the duplicate nav bar is not showing at all so I couldn't even use the inspect tool. Please help me out with this @varshith257

**Comment by varshith257** [comment]
See here

File: news.html [hunk 1]
```diff
@@ -29,6 +29,7 @@
   <link href="glightbox/css/glightbox.min.css" rel="stylesheet">
   <link href="remixicon/remixicon.css" rel="stylesheet">
   <link href="swiper/swiper-bundle.min.css" rel="stylesheet">
+  <link rel="stylesheet" href="../Css-Files/news.css">
```
Comment (varshith257) [inline news.html:32]: Remove this line.Line 11 has same code

**Comment by samarsajad** [comment]
> 
> 
> See here

IK it is in the deploy preview but not in my local system. Even if i try fixing it, I wouldn't know if it is fixed cause this is not visisble in my local system. If you could merge this pull request and if the duplicate nav bar appears in the main, I'd raise a new issue for that cause as if now i am not able to fix this

File: news.html [hunk 2]
```diff
@@ -54,6 +416,30 @@
...
+          <a href="./Html-Files/Doctor Experience.html"><button>Find a Doctor</button></a>
+
+        </ul>
+      </nav>
+      </div>
+    </header>
+
           <img id="rd-logo" style="width: 90px; height: auto;" src="images/rd.png">
         </div>
         <nav class="nav_menu">
```
Comment (varshith257) [inline news.html:445]: Here's the duplicate code. See from line 422 to down has navbar code again same as above.

**Review by varshith257 — CHANGES_REQUESTED** [review]

**Comment by varshith257** [comment]
Also align hover me button center to card and make sure not stick with image

**Comment by samarsajad** [comment]
i will create a new pull request. things got a lil messed up @varshith257

**Comment by samarsajad** [comment]
created a new pull request #961  ptal @varshith257

**Pull request closed without merging**